StripeFinancialConnections/StripeFinancialConnections/Source/Analytics/FinancialConnectionsAnalyticsClient.swift

```diff
@@ -18,12 +18,12 @@ protocol FinancialConnectionsAnalyticsClientDelegate: AnyObject {
 
 final class FinancialConnectionsAnalyticsClient {
 
-    private let analyticsClient: AnalyticsClientV2
+    private let analyticsClient: AnalyticsClientV2Protocol
```

Comment:
This change allows us to dependency inject a mock analytics client in the unit tests. This should not cause any functional change to the implementation.